src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py

```diff
@@ -439,6 +433,41 @@ def __init__(self, file_path: FilePathType, verbose: bool = True):
                     values=[str(celltype_mapping[str(x[0])]) for x in celltype_info["label"][0][0][0]],
                 )
 
+        # Register a dummy recorder to write the recording device metadata if desired
+
+        session_data_file_path = session_path / f"{session_id}.session.mat"
+        if session_data_file_path.is_file():
+            from pymatreader import read_mat
+
+            ignore_fields = ["animal", "behavioralTracking", "timeSeries", "spikeSorting", "epochs"]
+            session_data = read_mat(filename=session_data_file_path, ignore_fields=ignore_fields)["session"]
+            extracellular_data = session_data["extracellular"]
+            num_channels = int(extracellular_data["nChannels"])
+            num_samples = int(extracellular_data["nSamples"])
+            sampling_frequency = int(extracellular_data["sr"])
+
+            # Create a dummy recording extractor
+            from spikeinterface.core.numpyextractors import NumpyRecording
+
+            traces_list = [np.empty(shape=(1, num_channels))]
+            channel_ids = [str(1 + i) for i in range(num_channels)]
+            dummy_recording_extractor = NumpyRecording(
+                traces_list=traces_list,
+                sampling_frequency=sampling_frequency,
+                channel_ids=channel_ids,
+            )
+
+            # Add the channel metadata
+            dummy_recording_extractor = add_channel_metadata_to_recoder(
+                recording_extractor=dummy_recording_extractor, folder_path=session_path
+            )
+
+            # Need a time vector for the recording extractor
+            last_spikes_frame = self.sorting_extractor.get_all_spike_trains()[0][0][-1]
```

Comment:
@CodyCBakerPhD 
This is the trick that I mentioned. Creat a time vector that is as large as you need to cast the spikes (which spikeinterface stores as frames) to seconds later in `add_units_table`.